### DISCUSSION TITLE
Livestream not available

### DISCUSSION THREAD

**Discussion by negesti** [goggle/plugin.video.srfplaytv#25] [Q&A] [answered]
I'm blind or there is no button for Livestreams. Are the Livestreams available in the latest addon version using KODI 21 (Omega)?

**Comment by goggle** [comment, marked as answer]
If you are looking for live TV, then no, this is not available, I do suggest to use `pvr.zattoo` if you are looking for this.
If you are looking for live streams of sports events, then yes, you should be able to find them under `Topics/Themen -> Sport -> Die nächsten Livestreams`.

**Comment by negesti** [comment]
Thanks. Maybe the Readme should be updated from "Livestreams" to "Sport Livestream (Themen / Sports / Die nächsten Livestreams)
